### DISCUSSION TITLE
2006/07/16/%E7%99%BE%E5%BA%A6%E4%B9%9F%E6%8E%A8%E5%87%BA-blog-%E6%9C%8D%E5%8A%A1%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#93] [Announcements]
# 2006/07/16/%E7%99%BE%E5%BA%A6%E4%B9%9F%E6%8E%A8%E5%87%BA-blog-%E6%9C%8D%E5%8A%A1%E4%BA%86

LabVIEW 的地址被我给占了

https://blog.qizhen.xyz/2006/07/16/%E7%99%BE%E5%BA%A6%E4%B9%9F%E6%8E%A8%E5%87%BA-blog-%E6%9C%8D%E5%8A%A1%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yi
2006年07月21日 21:40 编辑
唉，我只抢了一个virtualInstruments

回复
永青
2006年07月17日 09:08 编辑
强烈鄙视！！！！
除非阮大侠能为这个域名作出点贡献来

回复
